cvat-canvas/src/typescript/interactionHandler.ts

```diff
@@ -8,6 +8,11 @@ import Crosshair from './crosshair';
 import {
     translateToSVG, PropType, stringifyPoints, translateToCanvas,
 } from './shared';
+
+import { getCVATStore } from '../../../cvat-ui/src/cvat-store';
+import { switchBlockMode } from '../../../cvat-ui/src/actions/settings-actions';
+import { ActiveControl } from '../../../cvat-ui/src/reducers/interfaces';
```

Comment:
It is wrong architecture design. ``cvat-canvas`` should not know about ``cvat-ui`` at all. If you change storage, u need to do it in cvat-ui

Reply:
Done. State logic is move to cvat-ui